src/policy/CMSRucioPolicy/permission.py

```diff
@@ -232,17 +233,17 @@ def _get_rule_size(rules):
 
     # Check if the account is banned
     if has_account_attribute(account, 'auto_approve_banned', session=session):
-        return False
+        return PermissionResult(False, "The account is banned.")
 
     # Check if the rule is locked
     if kwargs['locked']:
-        return False
+        return PermissionResult(False, "The rule is locked.")
 
     if kwargs['lifetime'] is None:
-        return False
+        return PermissionResult(False, "lifetime argument is None.")
```

Comment:
I think this should be worded in passive voice with an explanation:
Something on the lines of "Rules with infinite (without) lifetime cannot be created under so and so activity".

Reply:
same thing applies to other messages in general.